nemo/collections/cv/modules/non_trainables/reshape_tensor.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+# =============================================================================
+# Copyright (c) 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+# Copyright (C) tkornuta, IBM Corporation 2019
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+__author__ = "Tomasz Kornuta"
+
+"""
+This file contains code artifacts adapted from the original implementation:
+https://github.com/IBM/pytorchpipe/blob/develop/ptp/components/transforms/reshape_tensor.py
+"""
+
+from nemo.backends.pytorch.nm import NonTrainableNM
+from nemo.core.neural_types import AxisKind, AxisType, NeuralType, VoidType
+from nemo.utils import logging
+from nemo.utils.configuration_error import ConfigurationError
+from nemo.utils.decorators import add_port_docs
+
+__all__ = ['ReshapeTensor']
+
+
+class ReshapeTensor(NonTrainableNM):
+    """
+    Class responsible for reshaping the input tensor.
```

Comment:
Can you maybe add a very short example of input/output into docstring?

Reply:
Good catch, done